students/luftsmeerflier/lesson04/mailroom/benefactors.py

```diff
@@ -0,0 +1,18 @@
+
+list = [
+	{
+		'first_name' : "John",
+		'last_name' : "Snipes",
+		'amount' : 300
+	},
+	{
+		'first_name' : "Olga",
+		'last_name' : "Snowden",
+		'amount' : 500
+	},
+	{
+		'first_name' : "Lola",
+		'last_name' : "Curtis",
+		'amount' : 450
+	}
+]
```

Comment:
you should be using a dictionary as a main data structure in this lesson.

example: `donors = {"Bob Smith": [100, 200, 300]}`